test/behavior/packed-struct.zig

```diff
@@ -446,3 +446,115 @@ test "optional pointer in packed struct" {
     const x = T{ .ptr = &n };
     try expect(x.ptr.? == &n);
 }
+
+test "nested packed struct field access test" {
```

Comment:
You'll need to skip the test for self-hosted native backends:

```suggestion
test "nested packed struct field access test" {
    if (builtin.zig_backend == .stage2_x86_64) return error.SkipZigTest;
    // and so on for other native backends
```